packages/@aws-cdk/aws-route53/lib/hosted-zone.ts

```diff
@@ -112,6 +113,41 @@ export class PublicHostedZone extends HostedZone {
   public addVpc(_vpc: ec2.IVpcNetwork) {
     throw new Error('Cannot associate public hosted zones with a VPC');
   }
+
+  /**
+   * Adds a delegation from this zone to a designated zone.
+   *
+   * @param delegate the zone being delegated to.
+   * @param opts     options for creating the DNS record, if any.
+   */
+  public delegateTo(delegate: PublicHostedZone, opts: ZoneDelegationOptions = {}): void {
```

Comment:
I wonder if this should be called `addDelegation`. I think that's more idiomatic (#1741)

Reply:
Deal.